proto/pulumi/plugin.proto

```diff
@@ -42,4 +42,25 @@ message PluginDependency {
 // to, this message is used so the provider can still be passed the engine address to communicate with.
 message PluginAttach {
     string address = 1; // the grpc address for the engine
+}
+
+message PackageParameterization {
+    string name = 1; // the parameterized package name.
+    string version = 2; // the parameterized package version.
+    bytes value = 3; // the parameter value for the parameterized package.
+}
+
+// PackageDependency is information about a package that a program may depend upon.
+message PackageDependency {
+    string name = 1;    // the name of the plugin.
+    string kind = 2;    // the kind of plugin (e.g., language, etc).
+    string version = 3; // the semver for this plugin.
+    string server = 4; // the URL of a server that can be used to download this plugin, if needed.
+
+    // a map of the checksums for the plugin, will be empty from old language runtimes. The keys should match
```

Comment:
```suggestion
    // a map of the checksums for the plugin, will be empty for old language runtimes. The keys should match
```

Reply:
Or I guess it's `from` as in `the response from an old language runtime` ?

Reply:
yeh, old (actually current) language hosts always send empty for this.